### PR TITLE
Add "version" to VS versions

### PR DESCRIPTION
**Pull request by gewarren**

### CODE REVIEW

File: docs/core/deploying/deploy-with-vs.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ Once you've finished debugging, you can publish your self-contained deployment:
 
 <!-- markdownlint-disable MD025 -->
 
-# [Visual Studio 15.6 and earlier](#tab/vs156)
+# [Visual Studio version 15.6 and earlier](#tab/vs156)
```
Comment (mairaw) [inline docs/core/deploying/deploy-with-vs.md]: we'd probably need to get this tab name approved first

File: docs/core/versions/remove-runtime-sdk-versions.md [hunk 2]
```diff
@@ -201,7 +201,7 @@ The parent directories for the SDK and runtime are listed in the output from the
 
 ## .NET Core Uninstall Tool
 
-The [.NET Core Uninstall Tool](../additional-tools/uninstall-tool.md) (`dotnet-core-uninstall`) lets you remove .NET Core SDKs and Runtimes from a system. A collection of options is available to specify which versions should be uninstalled.
+The [.NET Core Uninstall Tool](../additional-tools/uninstall-tool.md) (`dotnet-core-uninstall`) lets you remove .NET Core SDKs and runtimes from a system. A collection of options is available to specify which versions should be uninstalled.
```
Comment (mairaw) [inline docs/core/versions/remove-runtime-sdk-versions.md]: @thraka and I discussed that when Runtimes was the product, we'd capitalize it

File: docs/csharp/language-reference/compiler-options/langversion-compiler-option.md [hunk 3]
```diff
@@ -21,7 +21,7 @@ Causes the compiler to accept only syntax that is included in the chosen C# lang
 
 ## Arguments
 
- `option`  
+ `option`
```
Comment (mairaw) [inline docs/csharp/language-reference/compiler-options/langversion-compiler-option.md]: if you remove the trailing spaces here, you need to add something. Perhaps a new line?
```suggestion
 `option`
 
```

image

**Review by mairaw — APPROVED** [review]
LGTM. Left a couple of suggestions for you to consider.

File: docs/core/deploying/deploy-with-vs.md (hunk 1 above)
Comment (gewarren) [reply docs/core/deploying/deploy-with-vs.md]: Okay, I'll just revert this then.

File: docs/core/versions/remove-runtime-sdk-versions.md (hunk 2 above)
Comment (gewarren) [reply docs/core/versions/remove-runtime-sdk-versions.md]: So this is sort of like saying "lets you remove Visual Studios from a system"? That doesn't seem quite right.

Comment (mairaw) [reply docs/core/versions/remove-runtime-sdk-versions.md]: Not really. You can download .NET Core SDKs or you can download .NET Core Runtimes:
e.g. https://dotnet.microsoft.com/download/dotnet-core/3.1

**Review by Thraka — APPROVED** [review]
My files look fine. Thanks!

File: docs/csharp/language-reference/compiler-options/langversion-compiler-option.md (hunk 3 above)
Comment (Thraka) [reply docs/csharp/language-reference/compiler-options/langversion-compiler-option.md]: And the `\` can't be used either right? It causes a loc problem?

Comment (gewarren) [reply docs/csharp/language-reference/compiler-options/langversion-compiler-option.md]: `\` can be used, it's not a problem for loc anymore. In this case I just went with a new line though.

**Comment by gewarren** [comment]
Thanks for the reviews!

**Pull request merged**